### PR TITLE
morphsnakes: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/morphsnakes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - scipy
+
+test:
+  imports:
+    - morphsnakes
+
+about:
+  home: "https://github.com/pmneila/morphsnakes"
+  license: "BSD 3-clause"
```
Comment (chrisburr) [inline recipes/morphsnakes/meta.yaml]:
```suggestion
  license: BSD-3-Clause
```

File: recipes/morphsnakes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - setuptools
+    - python
+    - scipy
```
Comment (chrisburr) [inline recipes/morphsnakes/meta.yaml]: Is this needed in `host`?
```suggestion
```

Comment (tschoonj) [reply recipes/morphsnakes/meta.yaml]: Unfortunately yes... the entire module is imported in setup.py to get the version number...

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**